.gitignore

```diff
@@ -7,6 +7,10 @@
 # Ignore bundler config.
 /.bundle
 
+# Ignore floo files
+.floo
+.flooignore
```

Comment:
It's the pair-programming thingy that Sean and I used recently to do pairing on an issue.